std/concurrency.d

```diff
@@ -73,6 +41,39 @@ import std.range.primitives;
 import std.range.interfaces : InputRange;
 import std.traits;
 
+// Avoids writeln output during a normal test-suite run
+// Using writeln is very helpful for runnable unittest examples
+version(unittest)
+{
+    void writeln(T...)(T t){}
```

Comment:
I am not a huge fan of this workaround, but this allows us to use `writeln` in the `unittest` below and thus convert the documented example to a `unittest` without loosing out on features. Better ideas are, of course, welcome.